packages/components/src/site-thumbnail/index.tsx

```diff
@@ -57,14 +58,19 @@ export const SiteThumbnail = ( {
 					className={ loader }
 					style={ {
 						position: 'absolute',
+						width: '100%',
+						height: '100%',
+						alignItems: 'center',
+						justifyContent: 'center',
+						display: 'flex',
 					} }
 				>
 					{ children }
 				</div>
 			) }
 			{ src && ! isError && (
 				<img
-					className="site-thumbnail__image"
+					className={ removeMshotsDefaultLoader }
```

Comment:
We usually use the `classnames` library for this sort of logic (which is already imported in this file)

```suggestion
					className={ classnames( {
						'site-thumbnail__image': isLoading,
					} ) }
```

Reply:
I see we're doing the same sort of class name logic for the `site-thumbnail-loader` class above ...